doc/autogen/types/integer.rst

```diff
@@ -12,10 +12,17 @@
 
     Computes the bit-wise 'xor' of the two integers.
 
+.. spicy:operator:: integer::Cast enum cast<enum-type>(int)
+
+    Converts the value into an enum instance. The value does *not* need to
+    correspond to any of the target type's enumerator labels.
+
 .. spicy:operator:: integer::Cast enum cast<enum-type>(uint)
 
     Converts the value into an enum instance. The value does *not* need to
-    correspond to any of the target type's enumerator labels
+    correspond to any of the target type's enumerator labels. It must not
+    be larger that the maximum that a *signed* integer value can
+    represent.
```

Comment:
*signed* integer value of the same length
would be a better phrase. But is this constraint really going in?!? That an enum cast from uint8 can't have value 130.

Reply:
Oh, there's an import bit missing, it's supposed to be: "It must not be larger that the maximum that a *signed* 64-bit integer value can represent." So that shouldn't really be much of a limitation at all.